tests/integ/modin/frame/test_tz_convert.py

```diff
@@ -101,7 +102,7 @@ def test_tz_convert_negative(axis, level, copy, exception):
 
     with pytest.raises(exception):
         snow_df.tz_convert(
-            tz="UTC",
+            tz=tz,
```

Comment:
Maybe we can match the error message in all these negative tests with something like
```
pytest.raises(
            exception, match=".*method doesn't support \'tz={tz}\'"
):
```

Reply:
The issue is that the `NotImplementedError` here can be caused by different scenarios of unsupported input parameters, not just `tz`. That's why we only check for the error type and not the error message.